### PR TITLE
Filter empty include parameters and empty resources

### PR DESCRIPTION
**Pull request by faizalmy**

### CODE REVIEW

File: src/Document.php [hunk 1]
```diff
@@ -14,7 +14,14 @@ class Document implements JsonSerializable
 
     public function addIncluded($link)
     {
+<<<<<<< HEAD
```
Comment (kylestev) [inline src/Document.php]: oops, looks like you left a merge conflict in here :wink:

**Comment by kylestev** [comment]
You might want to squash these commits into a single commit as well

**Comment by tobscure** [comment]
Yep, needs to be squashed and PSR-2'd.

**Comment by vinkla** [comment]
@faizalmy Maybe you could open up a new PR now since we've added tests?

**Pull request closed without merging**